tests/integration/test_bayesian_optimization.py

```diff
@@ -73,7 +73,7 @@
                 ),
             ),
             (
-                10,
+                12,
```

Comment:
This is purely due to the BatchReparametrizationSampler picking different _eps values to before.